src/components/LightweightChart/LightweightChart.tsx

```
@@ -0,0 +1,226 @@
+import { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+export interface LightweightChartProps {
+  symbol: string;
+  width?: string | number;
+  height?: string | number;
+  onChartReady?: () => void;
+}
+
+export const LightweightChart: React.FC<LightweightChartProps> = ({
+  symbol,
```

ESLint

[error] 'symbol' is missing in props validation

(react/prop-types)